ecs/src/nox/ecs/EntityManager.cpp

```diff
@@ -698,3 +698,15 @@ nox::ecs::EntityManager::getCollection(const TypeIdentifier& identifier)
 
     return *collection;
 }
+
+void
+nox::ecs::EntityManager::setLogicContext(nox::logic::Logic* logicContext)
```

Comment:
Always makes use of the variable as a nox::logic::Logic instead of a plain nox::logic::IContext. Saving it as the later would require us casting several places in order to use it.